src/modules/layout/components/UserBottomNav.tsx

```diff
@@ -1,36 +1,45 @@
-import { pages, type AllPath } from "@/modules/config/pageConfig";
+import { pages, type PageId } from "@/modules/config/pageConfig";
 import styled from "@emotion/styled";
 import { type Icon } from "@phosphor-icons/react";
-import { Segmented, Typography, theme } from "antd";
-import { useEffect, useState } from "react";
+import { ConfigProvider, Segmented, Typography, theme } from "antd";
+import Link from "next/link";
 
-const UserBottomNav = () => {
-  const { home, basket, orders } = pages;
-  const [selected, setSelected] = useState<AllPath>(home.path);
+type UserBottomNavProps = {
+  nowPageId: PageId;
+};
 
-  useEffect(() => {
-    console.log("selected page: ", selected);
-  }, [selected]);
+const UserBottomNav: React.FC<UserBottomNavProps> = ({ nowPageId }) => {
+  const { home, basket, orders } = pages;
 
   return (
-    <StyledSegmented
-      defaultValue={selected}
-      onChange={(value) => {
-        setSelected(value as AllPath);
+    <ConfigProvider
+      theme={{
+        components: {
+          Segmented: {
+            itemSelectedBg: "#ffffff",
+            itemActiveBg: "#ffffff",
+            colorBgLayout: "#ffffff",
+            boxShadowTertiary: "none",
+          },
+        },
       }}
-      block
-      options={[home, basket, orders].map((page) => ({
-        label: (
-          <AutoIcon
-            Component={page.Icon}
-            label={page.label}
-            matchKey={page.path}
-            matchState={selected}
-          />
-        ),
-        value: page.path,
-      }))}
-    />
+    >
+      <StyledSegmented
+        defaultValue={nowPageId}
+        block
+        options={[home, basket, orders].map((page) => ({
+          label: (
+            <AutoIcon
+              Component={page.Icon}
+              label={page.label}
+              matchPageId={page.id}
+              nowPageId={nowPageId}
```

Comment:
Can be another name like "currentSelectedPage", "selectedPage", "currentPage" ?

Reply:
ummm matchPageId variable is Id of button to conditional styling use with nowPageId ( could rename to currentPageId ? )

matchPageId could rename to option Id , linkId , buttonId , etc.

butๆๆ if it work eork it wif